cmd/cli/cluster.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis"
+)
+
+func renameAnyTagsSubscriptionsKeyForwards(database moira.Database) error {
```

Comment:
The difference between most rename methods is key strings. It looks like a good place for creating common function: `func renameKey(database moira.Database, key string, newKey string)`

Reply:
Done